integration_tests/__tests__/failures.test.js

```diff
@@ -14,7 +14,8 @@ const runJest = require('../runJest');
 
 const dir = path.resolve(__dirname, '../failures');
 
-const normalizeDots = text => text.replace(/\.{1,}$/gm, '.');
+const normalize = text =>
+  text.replace(/\.{1,}$/gm, '.').replace(/\bassert:\d+:\d+/g, 'assert:1:1');
```

Comment:
what is this change for?

Reply:
`assert` can change from one version to another, and since now we re-inject the native module, line numbers can change, thus not making the snapshot to match. This ensures that line numbers are transformed into something consistent across Node versions.

Since I'm going to try make them disappear, that doesn't make sense anymore.